tests/debugging/test_uploader.py

```diff
@@ -44,16 +44,13 @@ def test_uploader_full_buffer():
     size = 1 << 8
     with ActiveBatchJsonEncoder(size=size, interval=0.5) as uploader:
```

Comment:
```suggestion
    with ActiveBatchJsonEncoder(size=size, interval=1) as uploader:
```
We see this test being flaky so we might find it beneficial to increase this interval

Reply:
increased and also changed the second wait with awake()